docs/production/tls.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Securing communication between cortex components with TLS"
```

Comment:
Please always `Cortex` with the upper case `C`.

_This comment applies across the whole file._